lib/PostEditorBlocks/WooCommerceBlocksIntegration.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
Similar comment here, could you please add the `declare(strict_types=1);` statement?